.gitignore

```diff
@@ -12,3 +12,5 @@ coverage-final.json
 coverage-final.lcov
 npm-debug.log
 yarn.lock
+/src/styles/dgrid.css.d.ts
```

Comment:
these seem rather specific... there has to be a better glob for this.  In widgets we appear to be checking these in.  Why did we want to exclude them here?